pkg/R/sparkR.R

```diff
@@ -110,9 +110,19 @@ sparkR.init <- function(
                 mainClass = "edu.berkeley.cs.amplab.sparkr.SparkRBackend",
                 args = as.character(sparkRBackendPort),
                 javaOpts = paste("-Xmx", sparkMem, sep = ""))
-  Sys.sleep(2) # Wait for backend to come up
+
+  cat("Waiting JVM bring up ...\n")
+  while(TRUE) {
+    if(!connExists(.sparkREnv)) {
```

Comment:
@shivaram  Yes, but `.sparkREnv$sparkRBackendPort` is used in `sparkR.stop`, so I think leave it here is necessary and safe. (Or move it into `connectBackend` ?)